src/redux/configureStore.js

```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'redux';
+import { addRemoveBook, initialState } from './books';
```

Comment:
- [ ] according to the requirement your project should Import `createStore`, `combineReducers`  and `applyMiddleware`  from redux.